xwiki-platform-core/xwiki-platform-appwithinminutes/xwiki-platform-appwithinminutes-ui/src/main/resources/AppWithinMinutes/Content.xml

```diff
@@ -44,14 +44,13 @@
     #set ($name = 'templateContent')
     #set ($editedDocument = $xwiki.getDocument("$stringtool.removeEnd($className, 'Class')Template"))
     ## Don't load the WYSIWYG editor when editing the class, because it's too heavy.
```

Comment:
```suggestion
```

Doesn't make sense to keep the comment.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3594/commits/4371e521d0183839094f7947c7ca8ef58c649620 :+1: